### PR TITLE
Creating missing LWCs and QAs

### PR DESCRIPTION
**Pull request by dbreese**
The project did not have LWCs for editContactRecord and editOpportunityRecord. Also, did not have quick actions pre-defined for Contat.create, Contact.edit, and Opportunity.create|edit|view.

@khawkins @sfdctaka @maliroteh-sf @AdamLiechty

### CODE REVIEW

**Review by sfdctaka — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .localdevserver
 .DS_Store
 node_modules
+__lwr_cache__
```
Comment (khawkins) [inline .gitignore:6]: What generates this? That's a fairly unexpected artifact in this repo, to me at least.

**Review by khawkins — APPROVED** [review]
The code seems fine to me. @AdamLiechty or @eric-romero am I right to assume the only reason for the missing LWCs and QAs is just that they weren't a part of the original PoC? I don't know of any reason we'd specifically _not_ want to create them, but figured I'd ask.

File: .gitignore (hunk 1 above)
Comment (maliroteh-sf) [reply .gitignore:6]: If you use LWR server to preview LWCs in a project, it creates this folder (similar to how `lwc-dev-server` would create a `.localdev` folder). I think someone was using our internal Browser Container build to preview LWCs from this repo.

**Comment by AdamLiechty** [comment]
@khawkins 👍  the original reason for excluding these LWCs was minimalism: Create the minimum number of examples from which to build a complete solution. But adding these better fulfills the goal of having a more unified, complete example from which to start. If a particular implementation does not need any of these quick actions or LWCs, they can easily be removed.

**Review by AdamLiechty — APPROVED** [review]

**Pull request merged**